src/ixdat/techniques/ec_ms.py

```diff
@@ -426,6 +429,156 @@ def siq_ecms_calibration_curve(
         else:
             return cal
 
+    # DECONVOLUTION METHODS BELOW
+    def grab_deconvoluted_current(
```

Comment:
These will move to the deconvolution Calculator's `calculate_series` method (#182).
That'll enable its use on an `MSMeasurement` (if someone wanted to do that) as alluded to in the docstring.